apps/examples/testcase/le_tc/kernel/tc_clock.c

```diff
@@ -27,6 +27,8 @@
 #include <time.h>
 #include <tinyara/time.h>
 #include <sys/time.h>
+#include <sys/ioctl.h>
+#include <tinyara/testcase_drv.h>
 #include "../../../../../os/kernel/clock/clock.h"
```

Comment:
Could you check whether this header is necessary or not?

Reply:
I deleted it. and other headers are now required and i will move the entire tc to the driver.